src/barcode_query/model.rs

```diff
@@ -17,3 +17,9 @@ pub struct ResultExisting {
 pub struct ResultDuplicated {
     bar_code: BarCode,
 }
+
+#[derive(Clone, Debug)]
+pub struct Message {
+    pub message_type: u8,
+    pub message_str: String,
```

Comment:
Why not `r#type` and `content` instead of `message_type` and `message_str`

Reply:
they are how Tam called them at the beginning and I keep that, not that important imo